smoke_tests/happypath/resume.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Resume to continue existing application
+
+  Background:
+    Given I show my environment
+    Then I visit "/"
+    And I should see "What do you want to do?"
+    When I click the "Appeal against a tax decision" link
+    Then I should see "Appeal against a tax decision"
```

Comment:
I think we can get rid of this as it's in another test
'Then I should see "Appeal against a tax decision"'